cstore_compression.c

```diff
@@ -0,0 +1,165 @@
+/*-------------------------------------------------------------------------
+ *
+ * cstore_compression.c
+ *
+ * This file contains compression/decompression functions definitions
+ * used in cstore_fdw.
+ *
+ * Copyright (c) 2016, Citus Data, Inc.
```

Comment:
Since we get `CStoreCompressHeader ` definition from older postgres, should we add `Portions Copyright`? I really don't know this, but asking if it is necessary or not.